sheets/1/exercise-3.tex

```diff
@@ -11,160 +11,161 @@
+\begin{Infobox}
+	
+	\subsubsection*{Kommentare:}
+	Kommentare sind Teile des Codes, die vom Computer ignoriert werden.
+	Sie werden genutzt, um anderen (aber auch euch selbst) kenntlich zu machen, was im Code passiert, was noch verändert werden sollte oder um allgemein Anmerkungen zu hinterlassen.
+\end{Infobox}
 
 \begin{lstlisting}[title=\textbf{Kommentar Beispiel}]
-// Dies ist ein einfacher Kommentar über eine Zeile.
-// Er wird mit // gestartet und nimmt die gesamte Zeile hinter sich ein.
-someCode.run() //Kommentare können auch nach Operationen geschrieben werden.
-/*
- * Kommentare können sich auch über mehrere Zeilen erstrecken.
- * Diese beginnen mit /* und enden mit  */ . 
- * Um die Lesbarkeit zu verbessern, werden die Zeilen dazwischen meist 
- * mit einem * begonnen.
- */
+	// Dies ist ein einfacher Kommentar über eine Zeile.
+	// Er wird mit // gestartet und nimmt die gesamte Zeile hinter sich ein.
+	someCode.run(); //Kommentare können auch nach Operationen geschrieben werden.
+	/*
+	* Kommentare können sich auch über mehrere Zeilen erstrecken.
+	* Diese beginnen mit /* und enden mit  */ . 
+	* Um die Lesbarkeit zu verbessern, werden die Zeilen dazwischen meist 
+	* mit einem * begonnen.
+	*/
 \end{lstlisting}
 \lstset{
 	basicstyle=\small
 }
-\subsubsection*{Klassen:}
-Klassen sind Baupläne für Objekte. In diesen können Operationen und Attribute definiert werden, welche dann den Objekten zur Verfügung stehen.\\
-Eine Klasse beginnt immer mit dem \lstinline{class} Schlüsselwort, gefolgt von ihrem Namen. Diesen könnt ihr fast frei wählen, nur spezielle Schlüsselwörter wie zum Beispiel \lstinline{class} sind verboten. Anschließend folgt ein Paar geschweifter Klammern \{ \}, in denen der Inhalt der Klasse geschrieben wird.
-
-\begin{lstlisting}[title=\textbf{Klassen Syntax}]
-// < > wird immer durch das ersetzt was in < > beschrieben wird
-class <NameDerKlasse>{ // Beginn der Klasse
-	/*  
-	 * Inhalt der Klasse. Um die Lesbarkeit zu verbessern wird
-	 * Code innerhalb geschweifter Klammern eingerückt.
-	 */
-	<Typ> <Attributname>; //Definition eines Attributes.
+\begin{Infobox}
 	
-	// Definition einer Operation (Siehe nächsten Eintrag)
-	<Rückgabetyp> <OperationsName>(<Typ> <ParameterName>, ...){
-		// Inhalt der Operation
+	\subsubsection*{Klassen:}
+	Klassen sind Baupläne für Objekte.
+	In diesen können Operationen definiert werden, welche dann den Objekten zur Verfügung stehen.\\
+	Eine Klasse beginnt immer mit dem \lstinline{class} Schlüsselwort, gefolgt von ihrem Namen.
+	Diesen könnt ihr fast frei wählen, nur spezielle Schlüsselwörter wie zum Beispiel \lstinline{class} sind verboten.
+	Anschließend folgt ein Paar geschweifter Klammern \{ \}, in denen der Inhalt der Klasse geschrieben wird.
+	In Java muss der Dateiname gleich dem Klassennamen sein. Z.B. Klasse "Hund" $\rightarrow$ Datei "Hund.java".
```

Comment:
Für anführungszeichen gibt es den \q{text} befehl in unserer latex class. Für code verwenden wir \lstinline{code} für Dateipfade vielleicht \texttt